scipy/stats/_axis_nan_policy.py

```diff
@@ -4,12 +4,15 @@
 # that support `axis` and `nan_policy`, including a decorator that
 # automatically adds `axis` and `nan_policy` arguments to a function.
 
+import os
 import numpy as np
 from functools import wraps
 from scipy._lib._docscrape import FunctionDoc, Parameter
 from scipy._lib._util import _contains_nan, AxisError, _get_nan
+from scipy._lib._array_api import array_namespace, is_numpy
 import inspect
 
+_SCIPY_ARRAY_API = os.environ.get("SCIPY_ARRAY_API", False)
```

Comment:
Do you need the env variable here? If so, no need to pull from `os.environ` twice.

```suggestion
from scipy._lib._array_api import array_namespace, is_numpy, SCIPY_ARRAY_API
import inspect
```

Reply:
OK, I can pull from `scipy._lib._array_api` instead.

I was using `pytest.mark.skipif(SCIPY_ARRAY_API)` to skip tests that pass masked arrays, object arrays, etc.

Reply:
Done.